permissive-json-pointer/src/lib.rs

```diff
@@ -0,0 +1,772 @@
+#![doc = include_str!("../README.md")]
+
+use std::collections::HashSet;
+
+use serde_json::*;
+
+type Document = Map<String, Value>;
+
+const SPLIT_SYMBOL: char = '.';
+
+/// Returns `true` if the `subset` is contained in the `main` string.
```

Comment:
Could you change this description, improve it and use the names of the parameters?

Reply:
I improved the message :+1: